Selector/CommonSelectors.ts

```diff
@@ -0,0 +1,25 @@
+import { Page, Locator } from '@playwright/test';
+
+export default class CommonSelectors
+{
+    static async SearchBar(page : Page) : Promise<Locator>
+    {
+        return await page.locator(".DocSearch");
```

Comment:
Please move all framework functionalities to a util class where you should override it and add needed parameters.
EX: async clickElement(selector: string, nextSelector: string, selectorState: stateType)
1. selector will be the element to click on.
2. nextSelector will be the selector to wait for after the click.
3. selectorState will be the state that the func will wait for before clicking on it